### PR TITLE
docs: update config files

### PR DESCRIPTION
**Pull request by ild0tt0re**
# Description

Updated config files make clear that the firebase name is relative to the project ID and added missing env variables

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ild0tt0re** [comment]
@rhahao 
`CROWDIN_PROJECT_NAME` used in `src/v2/utils/announcement-utils.js`. Should we keep it?
`CROWDIN_PROJECT_ID` not used in the app? 

should we remove `CROWDIN` setup from the local setup guide?
Thanks!

**Comment by rhahao** [comment]
Hi @ild0tt0re, we no longer need the Crowdin setup so these can be removed. Thanks.

File: .env.example [hunk 1]
```diff
@@ -12,3 +12,4 @@ MAIL_SENDER_NAME=
 MAIL_PASSWORD=
 SEC_ENCRYPT_KEY=
 SOURCE_MATERIALS_API=https://api-source-materials.sws2apps.com/api
+USER_PARSER_API_KEY=
```
Comment (rhahao) [inline .env.example:15]: @ild0tt0re: we no longer use this `USER_PARSER_API_KEY` variable. Could you please remove it? Thank you.

**Review by rhahao — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (ild0tt0re) [reply .env.example:15]: Ok, can i remove user parser setup also from local development setup guide?

Comment (rhahao) [reply .env.example:15]: @ild0tt0re: I’ve already removed it on the guide. We will need to remove it from here. Thanks.

**Pull request merged**